src/coniql/strawberry_schema.py

```diff
@@ -0,0 +1,370 @@
+import asyncio
+import base64
+import datetime
+import json
+from enum import Enum
+from typing import List, Optional, Sequence
+
+import numpy as np
+import strawberry
+
+from coniql.caplugin import CAPlugin
+from coniql.plugin import PluginStore
+from coniql.pvaplugin import PVAPlugin
+from coniql.simplugin import SimPlugin
+
+store_global = PluginStore()
+store_global.add_plugin("ssim", SimPlugin())
+store_global.add_plugin("pva", PVAPlugin())
+store_global.add_plugin("ca", CAPlugin(), set_default=True)
+
+
+@strawberry.enum
+class NumberType(Enum):
+    """
+    Numeric type for arrays of numbers
+    """
+
+    INT8 = "INT8"
+    UINT8 = "UINT8"
+    INT16 = "INT16"
+    UINT16 = "UINT16"
+    INT32 = "INT32"
+    UINT32 = "UINT32"
+    INT64 = "INT64"
+    UINT64 = "UINT64"
+    FLOAT32 = "FLOAT32"
+    FLOAT64 = "FLOAT64"
+
+
+@strawberry.enum
+class ChannelRole(Enum):
+    """
+    What access role has the Channel
+    """
+
+    RO = "RO"
+    WO = "WO"
+    RW = "RW"
+
+
+@strawberry.enum
+class Widget(Enum):
```

Comment:
I have been able to do this so that I have just one copy that I re-use. See commit: https://github.com/dls-controls/coniql/pull/44/commits/1670d3ae80e85172631e795f042124c087210d4a.